network-api/networkapi/wagtailpages/utils.py

```diff
@@ -188,3 +188,22 @@ def get_content_related_by_tag(page, result_count=3):
         capture_exception(err)
 
     return result_list[:result_count]
+
+
+def insert_panels_after(panels, more, after_label):
```

Comment:
I'm not married to this signature, would `insert_panels_after(panels, label, more)` make more sense?